src/Error.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * **PostCSS Syntax Error**
+ *
+ * Loader wrapper for postcss syntax errors
+ *
+ * @class SyntaxError
+ * @extends Error
+ *
+ * @param {Object} err CssSyntaxError
+ */
+class SyntaxError extends Error {
+  constructor (err) {
+    super(err)
+
+    const { line, column, reason } = err
+
+    this.name = 'SyntaxError'
+
+    this.message = ''
+    this.message += `${this.name} \n\n(${line}:${column}) ${reason}`
```

Comment:
Line and column can be undefined if you don't use source map https://github.com/postcss/postcss/blob/8157278c743564555a22da67da2b5972945462d8/lib/input.es6#L124

Reply:
@michael-ciniawsky also any warning can be `SyntaxError` https://github.com/postcss/postcss/blob/master/lib/warning.es6#L76 and https://github.com/postcss/postcss/blob/8157278c743564555a22da67da2b5972945462d8/lib/node.es6#L84

Reply:
In fact, I do not understand why you asked so much about how they should look if you have not studied the source code before :confused: